operations/rack_resiliency/manage_critical_services.md

```diff
@@ -0,0 +1,316 @@
+    },
+    "coredns": {
+      "namespace": "kube-system",
+      "type": "Deployment"
+    },
+    "cray-power-control-bitnami-etcd": {
+      "namespace": "services",
+      "type": "StatefulSet"
+    },
+    "cray-powerdns-manager": {
+      "namespace": "services",
+      "type": "Deployment"
+    },
+    "cray-sls": {
+      "namespace": "services",
+      "type": "Deployment"
+    },
+    "cray-spire-postgres": {
+      "namespace": "spire",
+      "type": "StatefulSet"
+    },
+    ...
+}
+...
+```
+
+### 3. Delete the critical service from the ConfigMap
+
+#### 3.1 Look for the critical service(s) to be deleted 
+
+Look inside the `critical-service-config.json` field, under the `data` section.
+
+```bash
+kubectl edit configmap rrs-mon-static -n rack-resiliency
+```
+
+**Example output:**
+
+```yaml
+apiVersion: v1
+data:
+  ceph_monitoring_polling_interval: "60"
+  ceph_monitoring_total_time: "600"
+  ceph_pre_monitoring_delay: "60"
+  critical-service-config.json: |-
+    {
+      "critical_services": {
+        "cilium-operator": {
+          "namespace": "kube-system",
+          "type": "Deployment"
+        },
+        "coredns": {
+          "namespace": "kube-system",
+          "type": "Deployment"
+        },
+        "cray-activemq-artemis-operator-controller-manager": {
+          "namespace": "dvs",
+          "type": "Deployment"
+        },
+        "cray-capmc": {
+          "namespace": "services",
+          "type": "Deployment"
+        },
+        "cray-ceph-csi-cephfs-provisioner": {
+          "namespace": "ceph-cephfs",
+          "type": "Deployment"
+        },
+        "cray-ceph-csi-rbd-provisioner": {
+          "namespace": "ceph-rbd",
+          "type": "Deployment"
+        },
+    ...
+```
+
+#### 3.2 Remove the entire critical service block, including the trailing comma
+
+**Example of removing a critical service**
+
+For this example, this is the initial critical services list in the ConfigMap:
+
+```json
+{
+    "critical_services": {
+        "cilium-operator": {
+            "namespace": "kube-system",
+            "type": "Deployment"
+        },
+        "coredns": {
+            "namespace": "kube-system",
+            "type": "Deployment"
+        },
+        "cray-ceph-csi-rbd-provisioner": {
+            "namespace": "ceph-rbd",
+            "type": "Deployment"
+        },
+    }
+}
+```
+
+In this example, the administrator wishes to delete the `coredns` critical service. To do this, they remove the following three lines:
+
+```json
+"coredns": {
+    "namespace": "kube-system",
+    "type": "Deployment"
+},
+```
+
+After deleting the entry, the list will look like:
+
+```yaml
+{
+    "critical_services": {
+        "cilium-operator": {
+            "namespace": "kube-system",
+            "type": "Deployment"
+        },
+        "cray-ceph-csi-rbd-provisioner": {
+            "namespace": "ceph-rbd",
+            "type": "Deployment"
+        },
+    }
+}
+```
+
+#### 3.4 Important notes
```

Comment:
This is not a step in the procedure, so it shouldn't be numbered. Also, putting this at the end is not ideal if it is giving them reminders of what they should be doing on the earlier steps.